Cargo.toml

```diff
@@ -34,7 +34,7 @@ dbus-crossroads = "0.5.0"
 winit = "0.26.1"
 
 [target.'cfg(target_os = "windows")'.dev-dependencies]
-raw-window-handle = "0.5.0"
+raw-window-handle = "0.4.2"
```

Comment:
Why not use `0.5.0`?

Reply:
`0.5.0` has some breaking changes, it need to be consistent with `winit`.

https://github.com/rust-windowing/winit/blob/430a49ebc2e1dbe7a8e06518e52bd5de7f33f2e6/Cargo.toml#L48